src/test/java/org/broadinstitute/hellbender/tools/copynumber/CallCopyRatioSegmentsIntegrationTest.java

```diff
@@ -29,5 +30,13 @@ public void testCallSegments() {
         Assert.assertEquals(calledCopyRatioSegments.getMetadata(), copyRatioSegments.getMetadata());
         Assert.assertEquals(calledCopyRatioSegments.getIntervals(), copyRatioSegments.getIntervals());
         Assert.assertEquals(calledCopyRatioSegments.getRecords().stream().map(s -> s.getCall().getOutputString()).toArray(), new String[] {"+", "-", "0", "0"});
+
+        // Test writing the legacy format.  Note that reading cannot be done through the CNV tools, since the header has been stripped away.
+        final File legacySegmentFile = new File(outputFile.getAbsolutePath() + CallCopyRatioSegments.IGV_COMPATIBLE_FILE_SUFFIX);
```

Comment:
Clean up this path once the file suffix is changed.

Reply:
Thanks for reminder....  Done